platform/ontology/api_test.go

```diff
@@ -176,3 +176,51 @@ func TestNormalize(t *testing.T) {
 		}
 	}
 }
+
+func TestAPI_CurrentBlockNumber(t *testing.T) {
```

Comment:
Don't do integration/functional tests inside the unit test files.

You can create another file and use another build tag, like that:
https://github.com/trustwallet/blockatlas/blob/43b998bedcf11417cdc31601117335b408ceb45b/pkg/integration/integration_test.go#L1

Reply:
the main reason I write it here is that integration tests are ignored at pipeline